go.mod

```diff
@@ -2,27 +2,60 @@ module github.com/kubeapps/kubeapps
 
 go 1.16
 
-replace github.com/docker/docker => github.com/docker/docker v0.0.0-20190731150326-928381b2215c
+replace (
+	// required by Oras https://github.com/deislabs/oras/blob/main/go.mod
+	github.com/docker/distribution => github.com/docker/distribution v2.7.1+incompatible
+	github.com/docker/docker => github.com/moby/moby v20.10.6+incompatible
+
+	// k8s.io/kubernetes is not intended to be used as a module, so versions are not being properly resolved.
+	// This replacement is required, see https://github.com/kubernetes/kubernetes/issues/79384
+	// As we support new k8s versions, this replacements should be also updated accordingly.
```

Comment:
Having a closer look at the deps, I realized of two issues:

- we were stuck to k8s/kubernetes 0.13 version, which was introducing some security problems (see our Snyk).
- our oras version is conflicting with the one required by helm, so it cannot be upgraded

The solution is far from being elegant, but since the version cannot be resolved automatically, both `go mod -u` and dependabot are unable to update these deps

Reply:
Urg, yeah. And how are these replacements kept up to date?

Reply:
They don't (automatically), however, whenever we got a new version of the k8s go modules (and we want to upgrade), it is just a matter of changing every dep at the same time.
Fortunately, the versioning is very consistent: 0.xx.yy for the go modules and 1.xx.yy for the main k8s.io/kubernetes.

If they ever publish this dep as a module, we'll be able to remove these replacements.

Regarding the other, when Helm go package got updated, I guess they will be using the same Oras version as we currently use and there will be no need of having the replc.

But, the alternative was not being able to upgrade helm and k8s, a thing that was introducing about 20 vulnerabilities.
However, not a go dep expert at all, if you have any better solution I'm not aware of, I'll happily send another PR for fixing it.